### PR TITLE
Updated/Added JA files

### PR DESCRIPTION
**Pull request by Woven-YNozaki**
Could you review Japanese files?
Refrected [PR #8](https://github.com/ms1-learner/python-01-en/pull/8) in the EN repository.

### CODE REVIEW

File: 02_creating_python_files_and_control_flow/03_challenge/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# チャレンジ3
+
+ユーザー入力として文字列を受け取り、文字列に含まれる母音 (a, e, i, o, u) の数を数えるPythonプログラムを `count_vowels.py` に書いてください。大文字と小文字の区別は無視してください。
+
+出力の例:
+
+入力された文字列: Hello world!  
+母音の数: 3
```
Comment (dancer-yuma) [inline 02_creating_python_files_and_control_flow/03_challenge/README.md:8]: The input and output should remain in English.

**Review by dancer-yuma — CHANGES_REQUESTED** [review]
@Woven-YNozaki 
I dropped one comment!

File: 02_creating_python_files_and_control_flow/03_challenge/README.md (hunk 1 above)
Comment (Woven-YNozaki) [reply 02_creating_python_files_and_control_flow/03_challenge/README.md]: @dancer-yuma
Oh! Reverted these output samples into English. Thanks!

**Review by dancer-yuma — APPROVED** [review]
@Woven-YNozaki 
Looks good to me!

**Pull request merged**